README.md

```diff
@@ -47,7 +47,8 @@ Note that Home Assistant includes a component (`emulated_hue`) to communicate wi
         * Select "Choose an existing role", and underneath, select `lambda_basic_execution` if it exists
         * If `lambda_basic_execution` doesn't exist, select "Create a custom role" instead, and enter `lambda_basic_execution` as the "Role Name"
     * Leave the rest of the defaults alone, and click "Next"
-    * Check "Enable event source"
+    * Using AWS CLI run the following command: `aws lambda add-permissions --function-name [name of your function, for example haaska] --statement_id 1 --action lambda:InvokeFunction --principal alexa-appkit.amazon.com --event-source-token [your skill ID]`
```

Comment:
In my testing of this, I found the following:
- `add-permissions` should be `add-permission`
- `--statement_id` should be `--statement-id`

Reply:
Thanks.